fractal_server/app/api/v1/project.py

```diff
@@ -300,6 +300,24 @@ async def apply_workflow(
         workflow_id=workflow_id,
         **apply_workflow.dict(),
     )
+
+    stm = (
```

Comment:
I guess this check should go below the definition of `job = ApplyWorkflow(...`, right? It's true that what really matters is that the check is before the add/commit, but there is no need to create the job object if the check fails.